### PR TITLE
Refactor listing page

### PR DESCRIPTION
**Pull request by AseelRousan-op**
### Description
Refactor Listing Page
- Fixed shared button cursor on hover
- Remove unneeded styles 
- Refactored some styles

### CODE REVIEW

File: src/components/listing/Slider/index.jsx [hunk 1]
```diff
@@ -51,7 +50,7 @@ function PrevArrow({ onClick }) {
  * Listing page slider section.
  *
  * @param {Object} props        The component props.
- * @param {Object} props.images The component data.
+ * @param {Object} props.images Listing page slider images.
```
Comment (MZeq) [inline src/components/listing/Slider/index.jsx:53]:
```suggestion
 * @param {Object[]} props.images Listing page slider images.
```

This should be an array of objects

File: src/components/listing/Slider/index.jsx [hunk 2]
```diff
@@ -51,7 +50,7 @@ function PrevArrow({ onClick }) {
...
  *
  * @return {JSX.Element}
```
Comment (MZeq) [inline src/components/listing/Slider/index.jsx:55]: In here can you please change the breakpoint in the settings object to be 
```
breakpoint: breakpoints.desktop,
```

you would need to import the breakpoints at the top
```
```

File: src/components/listing/SimilarListings/index.jsx [hunk 3]
```diff
@@ -68,7 +68,7 @@ function SimilarListings({ similarListings }) {
             /* To Do */
```
Comment (MZeq) [inline src/components/listing/SimilarListings/index.jsx:68]: In the documentation above for this component
```
/**
 * Home page featured listing section.
 *
 * @param {Object}   props                 The component props.
 * @param {Object[]} props.similarListings The listings.
 *
 * @return {JSX.Element}
 */
```

**Review by MZeq — APPROVED** [review]
LGTM, general notes that can be addressed in different PRs I believe we should not use any hard coded breakpoint in the application (mostly used in the sliders).

And descontructing the props object in the functional components parameters, instead of having it to be in the first line in the components.

File: src/components/listing/ContactForm/contact-form.styles.js [hunk 4]
```diff
@@ -7,12 +7,11 @@ export const MainContainer = styled.div`
   border-radius: 0.56rem;
```
Comment (AhmadAlShwaiki) [inline src/components/listing/ContactForm/contact-form.styles.js:7]:
```suggestion
  border-radius: 0.5rem;
  border: 0.06rem solid ${colors.midGrey};
  margin-top: 1rem;
  height: 100%;
  padding: 1.5rem 2rem;


```

Comment (AhmadAlShwaiki) [inline src/components/listing/ContactForm/contact-form.styles.js:7]:
```suggestion
export const Container = styled.div`
  border-radius: 0.5rem;
  border: 0.06rem solid ${colors.midGrey};
  margin-top: 1rem;
  height: 100%;
  padding: 1.5rem 2rem;

  ${mq.desktop`
    margin-top: 0;
    min-width: 25rem;
  `};
`;
```

File: src/components/listing/ContactForm/contact-form.styles.js [hunk 5]
```diff
@@ -21,7 +20,7 @@ export const UserInfo = styled.div`
   align-items: center;
   display: flex;
   flex-direction: column;
-  margin: 0 0 1.68rem 0;
+  margin-bottom: 1.68rem;
```
Comment (AhmadAlShwaiki) [inline src/components/listing/ContactForm/contact-form.styles.js:23]:
```suggestion
  margin-bottom: 1.5rem;
```

Comment (AhmadAlShwaiki) [reply src/components/listing/ContactForm/contact-form.styles.js:23]: Below UserImg should be 
```export const UserImg = styled.img`
  border-radius: 50%;
  margin: 0 1.5rem 0 0;
`;
```

File: src/components/listing/ContactForm/contact-form.styles.js [hunk 6]
```diff
@@ -68,7 +67,8 @@ export const Input = styled.input`
   border-bottom: 0.06rem solid ${colors.midGrey};
   margin: 2.75rem 0 0 0;
   min-width: 100%;
-  outline: none;
+  font-size: 1.125rem;
```
Comment (AhmadAlShwaiki) [inline src/components/listing/ContactForm/contact-form.styles.js:70]:
```suggestion
export const Input = styled.input`
  border: none;
  border-bottom: 0.06rem solid ${colors.midGrey};
  min-width: 100%;
  font-size: 1rem;
  margin-top: 2.5rem;
  padding: 0.5rem 0 0.5rem 0;

  &:focus {
    border-color: ${colors.green};
    outline: none;
  }
`;
```

File: src/components/listing/ContactForm/contact-form.styles.js [hunk 7]
```diff
@@ -79,7 +79,6 @@ export const Message = styled.textarea`
   border: 0.06rem solid ${colors.midGrey};
   min-height: 7rem;
   min-width: 100%;
-  outline: none;
```
Comment (AhmadAlShwaiki) [inline src/components/listing/ContactForm/contact-form.styles.js:82]: ```export const Message = styled.textarea`
  border: 0.06rem solid ${colors.midGrey};
  min-height: 9.5rem;
  min-width: 100%;
  resize: none;
  font-size: 1rem;
  font-family: Montserrat;

  &:focus {
    border-color: ${colors.green};
    outline: none;
  }
`;
```

File: src/components/listing/Slider/index.jsx [hunk 8]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/prop-types */
```
Comment (Mahmoudzaid2994) [inline src/components/listing/Slider/index.jsx:1]:
```suggestion

```

**Review by AhmadAlShwaiki — APPROVED** [review]
Looks good, pushed a commit here, please recheck styles

**Pull request merged**